docs/sources/clients/promtail/scraping.md

```diff
@@ -4,6 +4,17 @@ description: Promtail Scraping (Service Discovery)
 ---
 # Scraping
 
+Promtail currently supports scraping from the following sources:
+- Files
+- Kubernetes Discovery
```

Comment:
```suggestion
- {{< relref "#kubernetes-discovery" >}}
```

Reply:
Hmmm, I'm looking at the right hand navigation/table of contents.  Is `Kubernetes Discovery` at the correct heading level?  Should it be under `File Target Discovery` or at the same level?